priv/perf/Makefile

```
@@ -0,0 +1,19 @@
+.PHONY: list
+list:
+	@$(MAKE) -pRrq -f $(lastword $(MAKEFILE_LIST)) : 2>/dev/null | awk -v RS= -F: '/^# File/,/^# Finished Make data base/ {if ($$1 !~ "^[#.]") {print $$1}}' | sort | egrep -v -e '^[^[:alnum:]]' -e '^$@$$'
+
+services_up:
+	cd ../../ && \
+	SNAPSHOT=SNAPSHOT_MIX_EXIT_PERIOD_SECONDS_120 make init_test && \
```

```suggestion
	SNAPSHOT=SNAPSHOT_MIX_EXIT_PERIOD_SECONDS_120 make init-contracts && \
```

I'm hoping to deprecate `make init_test` in favor of a more specific name `make init-contracts` since now it's used outside tests too. `make init_test` is now just [an alias to `make init-contracts`](https://github.com/omisego/elixir-omg/blob/master/Makefile#L225). Thanks!